frontend/src/components/Button.test.js

```diff
@@ -0,0 +1,27 @@
+import { render } from '@testing-library/react'
+import 'jest-styled-components'
+import user from '@testing-library/user-event'
+import Button from './Button'
+
+const testProps = {
+    text: 'Click me',
+    onClick: jest.fn()
```

Comment:
Global mock function may lead to bugs in your tests, i think you can just use a no op `() => {}` here.